### ISSUE TITLE
Failed to set up

### ISSUE THREAD

**Issue by raul811** [jeroenterheerdt/HAsmartirrigation#324]
### What happened?

After the installing of the integration does not start and says set up fail

### How to reproduce

I don't have any.
Diagnostic file only the log file from homo assistant

### Relevant log output

```shell
Logger: homeassistant.config_entries
Source: config_entries.py:406
First occurred: 11:48:48 (3 occurrences)
Last logged: 13:19:53

Error setting up entry Smart Irrigation for smart_irrigation
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/smart_irrigation/__init__.py", line 64, in async_setup_entry
    store = await async_get_registry(hass)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/smart_irrigation/store.py", line 561, in async_get_registry
    return cast(SmartIrrigationStorage, await task)
                                        ^^^^^^^^^^
voluptuous.error.MultipleInvalid: expected float for dictionary value @ data['delta']
```

### Which version are you running?

v2023.12.2

### Diagnostics file

- [X] I have attached a diagnostics file

### Additional information

In my integration smart irrigation I don't have a diagnostic field

**Comment by ma-wagner** [comment]
Hi, same here. I get the error after HA 2024.1.5 update. Thanks for help.

**Comment by jeroenterheerdt** [comment]
thanks for reporting! should be fixed in 2024.1.0
